src/ButtonV2/ButtonV2.tsx

```diff
@@ -5,7 +5,7 @@ import { StyledButton } from './ButtonV2.styles';
 export interface ButtonV2Props extends ButtonHTMLAttributes<HTMLButtonElement> {
   size?: 'small' | 'standard';
   radius?: 'standard' | 'round';
-  variant?: 'primary' | 'secondary' | 'tertiary' | 'danger';
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'danger' | 'white';
```

Comment:
Can the variant be something that better indicates that is contains the NEW marker rather than just saying it is a white button?

Reply:
I think we shouldn't have the new inside the button, we already have a badge component that can be placed over it when required in each app - that way we won't be over complicating the button component.

Reply:
I have removed the NEW marker and added another component for the button using badge, I've left this option as 'white' but removed the extra story.